Solutions/Windows Security Events/Hunting Queries/Least_Common_Process_Command_Lines.yaml

```diff
@@ -12,21 +12,20 @@ requiredDataConnectors:
 tactics:
   - Execution
 query: |
-
   let starttime = todatetime('{{StartTimeISO}}');
   let endtime = todatetime('{{EndTimeISO}}');
-  let lookback = starttime - 7d;
+  let lookback = starttime - 7d;  
   let Allowlist = dynamic (['foo.exe', 'baz.exe']);
-  let Sensitivity = 5;
+  let Sensitivity = 5;  
   SecurityEvent
   | where TimeGenerated between(lookback..endtime)
   | where EventID == 4688 and NewProcessName !endswith 'conhost.exe'
   | extend ProcArray = split(NewProcessName, '\\')
   // ProcArrayLength is Folder Depth
-  | extend ProcArrayLength = arraylength(ProcArray)
+  | extend ProcArrayLength = array_length(ProcArray)
   | extend LastIndex = ProcArrayLength - 1
   | extend Proc = ProcArray[LastIndex]
   | where Proc !in (Allowlist)
-  | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated), TimesSeen = count(), HostCount = dcount(Computer), Hosts = makeset(Computer), UserCount = dcount(SubjectUserName), Users = makeset(SubjectUserName) by CommandLine
+  | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated), TimesSeen = count(), HostCount = dcount(Computer), Hosts = make_set(Computer,maxSize=500), UserCount = dcount(SubjectUserName), Users = make_set(SubjectUserName,maxSize=500) by CommandLine
   | where TimesSeen < Sensitivity
   | extend timestamp = StartTimeUtc
```

Comment:
Entity mappings

Reply:
@petebryan : We are adding all hosts and user in an array, so that's why I think the mapping is not done